style.css

```diff
@@ -10,14 +10,14 @@ body {
   justify-content: center;
   align-items: center;
   height: 4910px;
-  width: 375px 100vw;
+  width: 375%;
 }
 
 header {
   position: absolute;
   top: 0;
   height: 627px;
-  width: 375px 100vw;
+  width: 375%;
```

Comment:
- [ ] Good job trying to implement the changes suggested by the previous reviewer, :recycle: However, let me suggest some tips to get it right:
  - let's use `width:100%` and `width:100vw` here instead of `375%`, because it is relative to the screen width and not exceed. 
  - then we can use `%` for the width of all the other containers we have on the app